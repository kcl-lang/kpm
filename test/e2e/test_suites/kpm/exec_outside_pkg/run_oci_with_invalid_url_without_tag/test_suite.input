kpm run oci://invalid_url