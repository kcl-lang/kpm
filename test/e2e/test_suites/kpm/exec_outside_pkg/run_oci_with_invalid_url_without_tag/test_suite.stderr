repository 'invalid_url/' not found
